libraries/AP_Logger/LogStructure.h

```diff
@@ -409,6 +409,7 @@ struct PACKED log_AHRS {
     int32_t lat;
     int32_t lng;
     float q1, q2, q3, q4;
+    int8_t ekf;
```

Comment:
Is this information available in the EKF logging itself?

If not, this should be a `uint8_t`....

Reply:
No It's not available. The reason I've set it to int8_t is to set it to -1 if logging is not enabled.
But I'm rebasing now anyway.

Reply:
Rebased and almost re-written, please review.